dao/src/main/resources/db/migration/V82__scannerIssuesOrtRun.sql

```diff
@@ -0,0 +1,70 @@
+-- This migration script integrates the issues generated by the Scanner into the ORT run issues table.
+-- In contrast to the scripts for the Analyzer and Advisor issues, this is more tricky because the issues are
+-- assigned to scan summaries, and there is no direct link to a package the scan summary belongs to. Therefore, the
+-- match is made via the provenance which can point either to a source code artifact or a VCS repository.
+
+-- Handle scanner issues for artifact provenances.
+INSERT INTO ort_runs_issues
+("ort_run_id", "issue_id", "identifier_id", "worker", "timestamp")
+SELECT
+  sj.ort_run_id,
+  i.id,
+  scan_result_identifiers.identifier_id,
+  'scanner',
+  i."timestamp"
+FROM scan_results sr
+INNER JOIN scan_summaries ss on sr.scan_summary_id = ss.id
+INNER JOIN scanner_runs_scan_results srsr on srsr.scan_result_id = sr.id
+INNER JOIN scanner_runs sr2 on srsr.scanner_run_id = sr2.id
+INNER JOIN scanner_jobs sj on sr2.scanner_job_id = sj.id
+INNER JOIN scan_summaries_issues ssi on ssi.scan_summary_id = ss.id
+INNER JOIN issues i on ssi.issue_id = i.id
+INNER JOIN (
+  SELECT
+    sr.id AS scan_result_id,
+    artifact_provenances.identifier_id,
+    artifact_provenances.artifact_id
+  FROM
+    scan_results sr,
+    (SELECT *
```

Comment:
Could you use named columns instead of `*`, to ease the readability of the query ?
Same underneath.

Reply:
Hm, probably yes. But what is done here is actually creating a temporary view that joins the `package_provenances` table with the `remote_artifacts` table. Then, in the outer SELECT, some fields from this view are selected. When using named columns, this would mean to either list all the columns of the involved tables (which is quite a number) or only the subset that is selected (which is somehow redundant to the outer `SELECT` clause). I find the current variant indeed more readable.

Reply:
I thought about listing only the columns you need e.g. `url`, `hash_value` and `hash_algorithm` (and 4 in the second case). This would be a bit faster too.

However since it's code that is run only one time, I am fine with the way it is.